library/Stratosphere/Types.hs

```diff
@@ -1,14 +1,19 @@
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass    #-}
```

Comment:
Did your editor automatically change the formatting of the pragmas and the imports? I like having non-context-dependent whitespace. It makes diffs much more sane once we have a lot of pragmas and imports.